src/components/MainScene.vue

```diff
@@ -61,17 +59,96 @@ export default {
 
         buildScene () {
             this.scene.clearColor = new BABYLON.Color3(0.8, 0.8, 0.8);
-            var camera = new BABYLON.FreeCamera("camera", new BABYLON.Vector3(0, 0, -10), this.scene);
-            var light = new BABYLON.PointLight("light", new BABYLON.Vector3(10, 10, 0), this.scene);
-
-            var item = BABYLON.Mesh.CreateBox('box', 2, this.scene);
-
-            item.rotation.x = -0.2;
-            item.rotation.y = -0.4;
-
-            const boxMaterial = new BABYLON.StandardMaterial('material', this.scene);
-            boxMaterial.emissiveColor = new BABYLON.Color3(0, 0.58, 0.86);
-            item.material = boxMaterial;
+            this.scene.createDefaultCameraOrLight(true, true, true);
+            this.scene.createDefaultEnvironment();
+
+            this.$store.state.Entities.addEntity(this.scene, {
+                'name': 'box',
+                'type': 'Shape',
+                'shape': 'box',
+                'position': { 'x': -5, 'y': 0, 'z': 0 },
+                'rotation': { 'x': -0.2, 'y': -0.4, 'z': 0 },
+                'dimensions': { 'x': 3, 'y': 3, 'z': 3 },
+                'color': { 'r': 1, 'g': 0, 'b': 0 }
+            }, {});
```

Comment:
Shouldn't really use strings for property IDs in script. This and similar could be written...
```suggestion
            this.$store.state.Entities.addEntity(this.scene, {
                name: 'box',
                type: 'Shape',
                shape: 'box',
                position: { x: -5, y: 0, z: 0 },
                rotation: { x: -0.2, y: -0.4, z: 0 },
                dimensions: { x: 3, y: 3, z: 3 },
                color: { r: 1, g: 0, b: 0 }
            }, {});
```